bindings/matrix-sdk-ffi/src/client.rs

```diff
@@ -365,6 +374,15 @@ impl Client {
             }
         }
     }
+
+    fn process_api_client_error(&self, error_kind: ErrorKind) {
+        if let ErrorKind::UnknownToken { soft_logout } = error_kind {
```

Comment:
We discussed this in the team, and agreed that there is no clear use case for processing other error kinds in the kind of central way enabled by the broadcast channel. Can you update (and rename) the channel to only send `struct UnknownToken { soft_logout: bool }`?